config/jobParameters/CombineUnloaderJobParameterSetup.xml

```diff
@@ -17,10 +17,12 @@
 			<Values>
 				<Value name="UNLOAD_COMBINE">1</Value>
 				<Value name="UNLOAD_SILO_LOADER">2</Value>
+				<Value name="UNLOAD_CHOPPER">3</Value>
```

Comment:
Not a big fan of adding another option here, as this complicates the usage for the player.
Maybe instead of creating a new unload copper strategy, we could keep the combine one for both but 
add a context with dependcy injection depending on the found unload target(combine or chopper)?

Reply:
> instead of creating a new unload copper strategy, we could keep the combine one for both 

I get why you don't like that but I still think it would be cleaner to have separate strategies. Another option could be to create a strategy for idle unloaders and instantiate the combine or chopper strategy only after the unloader is called by the combine/chopper.

Reply:
I would go further and move the unloading code out as well, just my two cents tho. You would be adding a lot of if statements to combine if you combined them as there are some things you just don't want a chopper unload driver to do